space-framework/pkg/space-manager/provider.go

```diff
@@ -227,6 +236,59 @@ func (p *provider) processProviderWatchEvents() {
 	}
 }
 
+const (
+	SECRET_NS       = "default"
+	SECRET_DATA_KEY = "kubeconfig"
+)
+
+func (p *provider) createSpaceSecrets(space *spacev1alpha1apis.Space, spInfo spaceprovider.SpaceInfo) error {
```

Comment:
There is no deleteSecrets func. We have to delete secrets when Space object deleted.

Reply:
added